Src/firmware/mainf.cpp

```diff
@@ -188,7 +221,7 @@ static bool initROS() {
 
   // Executor
   RCCHECK(rclc_executor_init(&executor, &support.context,
-                             15 + RCLC_EXECUTOR_PARAMETER_SERVER_HANDLES,
+                             17 + RCLC_EXECUTOR_PARAMETER_SERVER_HANDLES,
```

Comment:
You added only 1 service so only 1 extra handle is needed (publishers don't need executor handles)

Reply:
Done